tests/e2e/graphql/temporary_procedures/mgps.py

```diff
@@ -0,0 +1,8 @@
+import typing
+
+import mgp
+
+
+@mgp.read_proc
+def components(context: mgp.ProcCtx) -> mgp.Record(versions=list, edition=str):
```

Comment:
This makes sense yes but how is that exactly called in these files, can you explain me maybe? I don't see any calls to dbms.components or anything I know I am missing something :smile:

Reply:
That is something that is getting called when the GraphQL Library initiates connection with Memgraph. It is sent automatically.